app/views/messages/_form.slim

```diff
@@ -5,7 +5,8 @@
   label.no-top-margin for="message_body"
     strong = f.label(:message_label_text, class: 'no-top-margin')
   span.usa-form-hint = t('helpers.label.message.dont_sent_hint') + " "
-  = link_to 'content that would violate federal rules', about_path
+  = link_to 'content that could violate federal rules.', about_path
```

Comment:
why not something like 

`= link_to I18n.t('helpers.label.message.dont_sent_hint_2`), about_path`